engine/resources/exec.go

```diff
@@ -182,7 +184,7 @@ func (obj *ExecRes) Watch() error {
 		cmd := exec.CommandContext(ctx, cmdName, cmdArgs...)
 		cmd.Dir = obj.WatchCwd // run program in pwd if ""
 		// ignore signals sent to parent process (we're in our own group)
-		cmd.SysProcAttr = &syscall.SysProcAttr{
+		cmd.SysProcAttr = &unix.SysProcAttr{
```

Comment:
shouldn't it be syscall.SysProcAttr here instead of unix.SysProcAttr? At least according to the unix documentation: https://godoc.org/golang.org/x/sys/unix#SysProcAttr

Reply:
I've tried reverting that part of the change and also the waitstatus parts. It still doesn't work.
So it's at least something *else* in the patch.